### PR TITLE
Picard 2.1.0

### PR DESCRIPTION
**Pull request by Lctrs**

### CODE REVIEW

File: org.musicbrainz.Picard.json [hunk 1]
```diff
@@ -16,9 +16,10 @@
         "--filesystem=xdg-music"
     ],
     "build-options": {
-        "cflags": "-O2 -g -pipe -fstack-protector-strong -D_FORTIFY_SOURCE=2",
-        "cxxflags": "-O2 -g -pipe -fstack-protector-strong -D_FORTIFY_SOURCE=2",
-        "ldflags": "-fstack-protector-strong -Wl,-z,relro,-z,now",
+        "cflags": "-O2 -g -fstack-protector-strong",
+        "cxxflags": "-O2 -g -fstack-protector-strong",
+        "cppflags": "-O2 -D_FORTIFY_SOURCE=2",
+        "ldflags": "-L/app/lib -fstack-protector-strong -Wl,-z,relro,-z,now",
```
Comment (bochecha) [inline org.musicbrainz.Picard.json]: Won't we get that automatically when we move to FDO//18.08?

File: org.musicbrainz.Picard.json [hunk 2]
```diff
@@ -4,8 +4,6 @@
     "runtime-version": "5.11",
     "sdk": "org.kde.Sdk",
     "command": "picard",
-    "rename-desktop-file": "picard.desktop",
```
Comment (bochecha) [inline org.musicbrainz.Picard.json]: Has the desktop file also been renamed?

Last I checked they were adding an appdata and renaming the icons only.

File: org.musicbrainz.Picard.json [hunk 3]
```diff
@@ -24,7 +22,7 @@
             "PYTHON": "/usr/bin/python3",
             "PIP": "/usr/bin/pip3"
         }
-    },	
+    },
```
Comment (bochecha) [inline org.musicbrainz.Picard.json]: The pedantic in me would prefer you made this change in its own commit. :stuck_out_tongue:

File: org.musicbrainz.Picard.json [hunk 4]
```diff
@@ -217,15 +215,7 @@
...
                     "url": "https://github.com/metabrainz/picard.git"
-                },
-                {
-                    "type": "file",
-                    "path": "org.musicbrainz.Picard.appdata.xml"
                 }
-            ],
-            "post-install": [
-                "mkdir -p /app/share/appdata",
-                "install -m0644 -p org.musicbrainz.Picard.appdata.xml /app/share/appdata/"
```
Comment (bochecha) [inline org.musicbrainz.Picard.json]: Hurray! :smile:

**Comment by Lctrs** [comment]
It's not meant to be merged now.

This is the PR for their next release.

File: org.musicbrainz.Picard.json (hunk 1 above)
Comment (Lctrs) [reply org.musicbrainz.Picard.json]: Yes. But we're not yet. :)

File: org.musicbrainz.Picard.json (hunk 2 above)
Comment (Lctrs) [reply org.musicbrainz.Picard.json]: They have.

File: org.musicbrainz.Picard.json (hunk 3 above)
Comment (Lctrs) [reply org.musicbrainz.Picard.json]: Too late I'm afraid. It's no big deal anyway imho.

Comment (bochecha) [reply org.musicbrainz.Picard.json]: It's never too late with git. :stuck_out_tongue:

File: org.musicbrainz.Picard.json (hunk 1 above)
Comment (bochecha) [reply org.musicbrainz.Picard.json]: Let's just wait for that, then.

File: org.musicbrainz.Picard.json (hunk 2 above)
Comment (bochecha) [reply org.musicbrainz.Picard.json]: Great!

**Comment by Lctrs** [comment]
@bochecha It's ready. Do you have any suggestion ?

**Comment by bochecha** [comment]
I'll try to get to it ASAP, but unfortunately I might not be able to quickly.

If you don't get any response from me in a week, use your own judgement and merge if you're happy with your changes. :wink:

**Comment by Lctrs** [comment]
I'm already happy with my changes. :D

**Pull request merged**